UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminLogs/AdminLogsWindow.cs

```diff
@@ -53,12 +55,20 @@ private void RequestAllLogFileNames()
 			RequestLogFilesNames.Send(new RequestLogFilesNames.NetMessage());
 		}
 
+
+		public void Search()
+		{
+			lastPageNumber = 1;
+			CurrentSelectedPageInput.text = 1.ToString();
+			RequestLogPage(logFilesDropdown.captionText.text, lastPageNumber);
+		}
+
 		public void UpdateLogFileDropdown(List<string> logFileNames)
 		{
 			Loggy.Info(logFileNames.Count.ToString());
 			logFilesDropdown.ClearOptions();
 			logFilesDropdown.AddOptions(logFileNames);
-			logFilesDropdown.value = logFilesDropdown.options.Count - 1;
+			logFilesDropdown.value = 0;
```

Comment:
what does this fix?

Reply:
I've invited list so the log that shop on the UI are the most recent ones near the top, but that means it needs to use the different index

Reply:
*inverted